src/bin/lib/render-migration.ts

```diff
@@ -19,8 +19,8 @@ const renderBatch = function (batch: RequestBatch) {
 }
 
 const renderPlan = (batches: RequestBatch[], envrionment: string) => {
-  console.log(chalk`{bold.underline Environment}: {bold.yellow ${envrionment}}\n`)
   console.log(chalk`{bold.green The following migration has been planned}\n`)
+  console.log(chalk`{bold.underline Environment}: {bold.yellow ${envrionment}}\n`)
```

Comment:
Looks like you have a typo here: `${envrionment}` 😉

Reply:
Haha, but thanks to auto-complete it's consistently wrong 😂